docs/layers.md

```diff
@@ -665,30 +665,57 @@ At zoom 12 and below, we consolidate some landuse kinds to reduce the amount of
 
 The current mappings are:
 
+* `airfield` -> `aerodrome`
 * `allotments` -> `urban_area`
+* `artwork` -> `urban_area`
+* `attraction` -> `urban_area`
 * `bare_rock` -> `desert`
-* `cemetery` -> `urban_area`
+* `cinema` -> `urban_area`
+* `college` -> `university`
 * `commercial` -> `urban_area`
+* `common` -> `grassland`
+* `container_terminal` -> `urban_area`
+* `cutline` -> `forest`
+* `dam` -> `land`
+* `danger_area` -> `military`
 * `farm` -> `farmland`
+* `ferry_terminal` -> `urban_area`
+* `fort` -> `urban_area`
+* `generator` -> `urban_area`
 * `grass` -> `grassland`
 * `heath` -> `grassland`
 * `industrial` -> `urban_area`
 * `meadow` -> `grassland`
 * `mud` -> `wetland`
 * `natural_wood` -> `forest`
 * `orchard` -> `farmland`
+* `pitch` -> `urban_area`
+* `place_of_worship` -> `urban_area`
+* `plant` -> `urban_area`
 * `plant_nursery` -> `farmland`
+* `port` -> `urban_area`
+* `port_terminal` -> `urban_area`
+* `prison` -> `urban_area`
 * `quarry` -> `barren`
+* `railway` -> `urban_area`
+* `range` -> `military`
 * `recreation_ground` -> `urban_area`
 * `residential` -> `urban_area`
 * `retail` -> `urban_area`
+* `rock` -> `barren`
 * `sand` -> `desert`
 * `school` -> `urban_area`
```

Comment:
Please do not merge `school`, it's more of an overlay.

Reply:
Sorry, I didn't catch the change in #1795 from https://github.com/tilezen/vector-datasource/issues/1779#issuecomment-452452746. Fixed in bc5de76.